config-templates/application-integration-test.properties.ctmpl

```diff
@@ -1,11 +0,0 @@
-{{- with $host := (( "http://34.68.70.164" )) -}}
-{{- with $wsmBaseUrl := (( printf "%s/dev-terra-workspace-manager" $host )) -}}
-{{- with $wsmCreateWorkspaceUrl := (( printf "%s/api/workspaces/v1" $wsmBaseUrl )) -}}
-{{- with $serviceAccountEmail := (( "william.thunderlord@test.firecloud.org" )) -}}
-{{- with $serviceAccountFilePath := env "SERVICE_ACCOUNT_FILE_PATH" -}}
-
-it.wsmCreateWorkspaceUrl={{$wsmCreateWorkspaceUrl}}
-it.serviceAccountEmail={{$serviceAccountEmail}}
-it.serviceAccountFilePath={{$serviceAccountFilePath}}
-
-{{end}}{{end}}{{end}}{{end}}{{end}}
```

Comment:
we'll have to put all the environment-specific code back in at some point (see https://github.com/DataBiosphere/terra-workspace-manager/pull/31/files#r431286413), I'm fine with whatever approach we take until we get to that point.

Reply:
Yeah; we thought that for the test resources, the environment selection should be able to happen at test run time rather than at render time.